scripts/create_and_deploy_private_version.sh

```diff
@@ -21,8 +21,8 @@
 
 set -e
 
-if [[ "${S3_UPLOAD_ID}" = "" || "${S3_UPLOAD_SECRET}" = "" || "${S3_UPLOAD_BUCKET}" = "" ]]; then
-    echo "You need to export S3_UPLOAD_ID, S3_UPLOAD_SECRECT and S3_UPLOAD_BUCKET for this to work"
+if [[ "${AWS_ACCESS_KEY_ID}" = "" || "${AWS_SECRET_ACCESS_KEY}" = "" || "${S3_UPLOAD_BUCKET}" = "" ]]; then
+    echo "You need to export AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and S3_UPLOAD_BUCKET for this to work"
```

Comment:
I think that when `-b` is provided we don't need S3_UPLOAD_BUCKET

Reply:
fixed.